changelog/12335.feat.md

```diff
@@ -0,0 +1 @@
+Add NLG endpoint `include_message` option to pass the domain message text to the NLG.
```

Comment:
This file name should be labeled `feature` instead of `feat`.

Also I thought the purpose of the NLG server is to store the response text, then this doesn't need to be provided under `responses` in the domain. It is unclear to me from this phrasing of the changelog entry what the change is about, or how is it beneficial.